### PR TITLE
fix: detect configuration before running tools

### PR DESCRIPTION
**Pull request by varl**
- chore: remove execute bit on source files
- fix: do not run tools without configs

### CODE REVIEW

File: src/commands/types/javascript.js [hunk 1]
```diff
@@ -50,24 +51,32 @@ exports.handler = (argv, callback) => {
...
 
-    if (finalStatus() === 0) {
-        log.print('All matched files pass the lint rules.')
-        log.print('')
+        if (finalStatus() === 0) {
+            log.print('All matched files pass the lint rules.')
+            log.print('')
+        }
+    } else {
+        log.debug('No ESLint configuration found')
```
Comment (mediremi) [inline src/commands/types/javascript.js:67]: Do we want to make this (and the prettier and ls-lint versions of this line) `log.warn` instead?

**Review by mediremi — APPROVED** [review]

File: src/commands/types/javascript.js (hunk 1 above)
Comment (mediremi) [reply src/commands/types/javascript.js:67]: I guess requiring all these tools would be fine internally but might not make sense for external developers :thinking:

Comment (varl) [reply src/commands/types/javascript.js:67]: I considered that a bit noisy for projects that don't want to use some of the tools.

We can change it to `log.log` which is visible with the `--verbose` flag, as opposed to the `--debug` flag which is tremendously chatty.

Comment (varl) [reply src/commands/types/javascript.js]: 478ab59

**Pull request merged**